src/screens/LoginScreen/index.js

```diff
@@ -29,26 +53,24 @@ const LoginScreen = ({ history }) => {
   }
 
   return (
-    <div className="container">
-      <Login>
-        <div className="jumbotron">
-          <h2>Login!</h2>
-          <form onSubmit={handleLogin}>
-            <div className="form-group">
-              <label>Email address</label>
-              <input name="email" type="email" className="form-control" />
-            </div>
-            <div className="form-group">
-              <label>Password</label>
-              <input name="password" type="password" className="form-control" />
-            </div>
-            <button type="submit" className="btn btn-primary">
-              Submit
-            </button>
-          </form>
-        </div>
-      </Login>
-    </div>
+      <Container>
+        <Login>
+            <h2 className="center">Login!</h2>
```

Comment:
a7san law component, even if it's a global component in "src/components" aw in "./styles", whatever